### PR TITLE
feat: fix testrail update script

### PR DESCRIPTION
**Pull request by refael-m**

### CODE REVIEW

File: src/commons/GitUtils.ts [hunk 1]
```diff
@@ -11,18 +11,22 @@ export namespace GitUtils {
   export function getLastMergedTestsIds(): Set<string> {
```
Comment (FelixZilber) [inline src/commons/GitUtils.ts:11]: check if we still need this function

File: src/commons/GitUtils.ts [hunk 2]
```diff
@@ -11,18 +11,22 @@ export namespace GitUtils {
...
-  export function getMergedTestsIdsFromDate(date: Date): Set<string> {
-    console.log(`Getting tests id's from date ${date}`);
-    const params: string = `-p master@{${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}}..master@{now}`;
-    const mergedTestsFiles: Set<string> = TestFilesUtils.getTestsFiles(executeGitScript(params));
+  export function getMergedTestsIdsSinceDay(sinceDay: number): Set<string> {
+    console.log(`Getting tests id's since ${sinceDay} days ago`);
+    const params: string = `whatchanged --since=@{${sinceDay}.days.ago} -p -m --first-parent --pretty=format: --name-only`;
+    const mergedTestsFiles: Set<string> = TestFilesUtils.getTestsFiles(
+      new Set<string>(executeGitScript(params).split(/[\r\n]+/))
+    );
```
Comment (FelixZilber) [inline src/commons/GitUtils.ts:29]: separate into 2 function calls(2 variables)

**Review by FelixZilber — CHANGES_REQUESTED** [review]

File: src/commons/GitUtils.ts (hunk 1 above)
Comment (refael-m) [reply src/commons/GitUtils.ts:11]: removed

File: src/commons/GitUtils.ts [hunk 3]
```diff
@@ -31,13 +22,13 @@ export namespace GitUtils {
...
   console.log(`Executing git command in: ${execSync(currGitUrlScript, { timeout: 30000 }).toString()}`);
   console.log(`Executing git command with ${params} param`);
-  const script: string = `git log ${params} --name-only --pretty="format:"`;
+  const script: string = `git ${params}`;
   console.log(`Executing: ${script}`);
   const res: string = execSync(script, { timeout: 30000 }).toString();
-  console.log(`Git command res: ${res ? res : 'No result returned!!!'}`);
-  return new Set<string>(res.split(/[\r\n]+/));
+  console.log(`Git command res: ${res ? res : 'No result!!!'}`);
+  return res ? res : '';
```
Comment (FelixZilber) [inline src/commons/GitUtils.ts:33]: don't you want to keep it undefined instead of empty string?

**Review by FelixZilber — APPROVED** [review]

File: src/commons/GitUtils.ts (hunk 3 above)
Comment (refael-m) [reply src/commons/GitUtils.ts:33]: done

File: src/commons/GitUtils.ts (hunk 2 above)
Comment (refael-m) [reply src/commons/GitUtils.ts]: done

**Pull request merged**